vroom/rubocop.vroom

```diff
@@ -0,0 +1,59 @@
+Rubocop is a linter and formatter for ruby.
+If you aren't familiar with basic codefmt usage yet, see main.vroom first.
+
+First, set up the vroom environment.
+
+  :source $VROOMDIR/setupvroom.vim
+
+  :let g:repeat_calls = []
+  :function FakeRepeat(...)<CR>
+  |  call add(g:repeat_calls, a:000)<CR>
+  :endfunction
+  :call maktaba#test#Override('repeat#set', 'FakeRepeat')
+
+  :call codefmt#SetWhetherToPerformIsAvailableChecksForTesting(0)
+
+By default, the rubocop executable is called. To use this plugin, rubocop
+must be installed on your system.
+  :FormatCode rubocop
+  ! rubocop .*
```

Comment:
Won't this fail for someone who doesn't have rubocop installed?

I could be wrong but I believe if you match a shell call like this but don't hijack to stub out any stdout or anything it'll still try to call the external command on your system. Prefer to stub out expected rubocop behavior so
  (a) if that behavior changes out from under us and breaks the plugin it's easier to understand what happened, and
  (b) we can run all the plugin's tests without needing to have dozens of different external tools installed.

Reply:
If you can uninstall rubocop and still get the test passing I'm probably just mistaken about how the shell hijacks work. It doesn't seem to choke on the some_other_program part like I'd expect (or maybe it does but the failure is too silent and doesn't bomb out of the whole test run?).

Reply:
I'm new to vroom, but my understanding is that system calls are all stubbed out by default. You can see it doing what you're asking down on line 26: it makes a call to a non-existent program. If you add command output, with $, and expectations, the dummy command works like all the other calls. I kept the test case minimal just to make it clearer what each case is testing.

Reply:
Alright, I had a closer look and it seems legit.